docs/testplan/Smartswitch-test-plan.md

```diff
@@ -11,12 +12,21 @@
     - [1.4 Check DPU console](#14-check-DPU-console)
     - [1.5 Check midplane ip address between NPU and DPU](#15-check-midplane-ip-address-between-npu-and-dpu)
     - [1.6 Check DPU shutdown and power up individually](#16-check-DPU-shutdown-and-power-up-individually)
-    - [1.7 Check removal of pcie link between NPU and DPU](#17-check-removal-of-pcie-link-between-npu-and-dpu)
+    - [1.7 Check pcie link status between NPU and DPU](#17-check-pcie-link-status-between-npu-and-dpu)
     - [1.8 Check the NTP date and timezone between DPU and NPU](#18-check-the-ntp-date-and-timezone-between-dpu-and-npu)
     - [1.9 Check the State of DPUs](#19-check-the-state-of-dpus)
     - [1.10 Check the Health of DPUs](#110-check-the-health-of-dpus)
     - [1.11 Check reboot cause history](#111-check-reboot-cause-history)
     - [1.12 Check the DPU state after OS reboot](#112-check-the-dpu-state-after-os-reboot)
+    - [1.13 Check memory on DPU](#113-check-memory-on-dpu)
+    - [1.14 Check DPU status and pcie Link after memory exhaustion on Switch](#114-check-dpu-status-and-pcie-link-after-memory-exhaustion-on-switch)
+    - [1.15 Check DPU status and pcie Link after memory exhaustion on DPU](#115-check-dpu-status-and-pcie-link-after-memory-exhaustion-on-dpu)
+    - [1.16 Check DPU status and pcie Link after restart pmon](#116-check-dpu-status-and-pcie-link-after-restart-pmon)
+    - [1.17 Check DPU status and pcie Link after reload of configuration](#117-check-dpu-status-and-pcie-link-after-reload-of-configuration)
+    - [1.18 Check DPU status and pcie Link after kernel panic on Switch](#118-check-dpu-status-and-pcie-link-after-kernel-panic-on-switch)
+    - [1.19 Check DPU status and pcie Link after kernel panic on DPU](#119-check-dpu-status-and-pcie-link-after-kernel-panic-on-dpu)
+    - [1.20 Check DPU status and pcie Link after power off reboot](#120-check-dpu-status-and-pcie-link-after-power-off-reboot)
```

Comment:
Instead, could you rephrase this way for clarity "1.20 Check DPU status and pcie Link after power cycle DPUs?"

Reply:
Sure, Changed it.
This is for switch power cycle only.

Reply:
Changed it and committed